web_responsive/views/web.xml

```diff
@@ -0,0 +1,242 @@
+
+            <t t-set="body_classname" t-value="'drawer drawer--left'" />
+
+            <header role="banner">
+                <nav id="odooAppDrawer" class="app-drawer-nav drawer-nav" role="navigation">
+                    <t t-call="web.menu" />
+                </nav>
+
+                <nav class="navbar navbar-default main-nav"
+                     role="navigation"
+                     groups="base.group_user,base.group_portal"
+                     id="oe_main_menu_navbar"
+                     >
+                    <div class="container-fluid">
+
+                        <div class="navbar-header">
+
+                            <a href="#"
+                               class="drawer-toggle navbar-collapse collapse btn btn-default app-drawer-toggle"
+                               accesskey="A"
+                               >
+                                <span class="sr-only">Toggle App Drawer</span>
+                                <i class="fa fa-th fa-lg app-drawer-icon-open" />
+                            </a>
+
+                            <button type="button"
+                                    class="app-drawer-toggle drawer-toggle pull-left navbar-toggle collapsed"
+                                    >
+                                <span class="sr-only">Toggle App Drawer</span>
+                                <div class="fa fa-th fa-lg app-drawer-icon-open" />
+                            </button>
+
+                            <button type="button"
+                                    id="odooMenuBarToggle"
+                                    class="navbar-toggle collapsed pull-right"
+                                    data-toggle="collapse"
+                                    data-target="#odooMenuBarNav"
+                                    >
+                                <span class="sr-only">Toggle Navigation</span>
+                                <i class="fa fa-bars fa-lg" />
+                            </button>
+
+                        </div>
+
+                        <div class="collapse navbar-collapse text-center"
+                             id="odooMenuBarNav"
+                             data-parent="#odooMenuBarToggle"
+                             aria-expanded="false"
+                             >
+                            <div class="oe_leftbar"
+                                 groups="base.group_user,base.group_portal"
+                                 >
+                                <t t-call="web.menu_secondary" />
+                            </div>
+                            <div class="nav navbar-nav navbar-right">
+                                <ul class="nav navbar-nav navbar-right oe_user_menu_placeholder"
+                                    style="display: none;"
+                                    />
+                                <ul class="nav navbar-nav navbar-right oe_systray"
+                                    style="display: none;"
```

Comment:
This is copied from Odoo core, and is required for their Javascript unfortunately